### PR TITLE
PatternFly milestone update

### PR DESCRIPTION
**Pull request by dlabrecq**
Updated @patternFly and r@edhat-cloud-services component packages to latest milestones.

In addition, I've refactored all @patternfly/react-token (and a couple remaining @patternfly/react-icons) imports to use full paths. This cuts our build time down by more than half; from 4 min 35 sec to approx 1 min 50 sec.

https://issues.redhat.com/browse/COST-416

### CODE REVIEW

File: src/pages/costModels/costModel/index.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
   TabContent,
   Title,
 } from '@patternfly/react-core';
-import { ErrorCircleOIcon } from '@patternfly/react-icons';
+import { ErrorCircleOIcon } from '@patternfly/react-icons/dist/js/icons/error-circle-o-icon';
```
Comment (boaz0) [inline src/pages/costModels/costModel/index.tsx:9]: why are we importing this from 'dist/js' and not from '@patternfly/react-icons'?

File: src/pages/costModels/costModelsDetails/components/markup.styles.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { global_FontSize_xl, global_spacer_md } from '@patternfly/react-tokens';
+import global_FontSize_xl from '@patternfly/react-tokens/dist/js/global_FontSize_xl';
```
Comment (boaz0) [inline src/pages/costModels/costModelsDetails/components/markup.styles.ts:1]: Also here, is this how we suppose to import?

File: src/pages/costModels/costModel/index.tsx (hunk 1 above)
Comment (dlabrecq) [reply src/pages/costModels/costModel/index.tsx:9]: By including exactly what we need, webpack doesn't need to filter thousands of tokens and icons. Webpack opens and parses thru patternfly's 3k+ files to find what we need. Each time we do that, it adds .5-1 sec to the build.

File: src/pages/costModels/costModelsDetails/components/markup.styles.ts (hunk 2 above)
Comment (dlabrecq) [reply src/pages/costModels/costModelsDetails/components/markup.styles.ts:1]: Yes. Please see the description above

**Comment by boaz0** [comment]
LGTM

**Pull request merged**